pony-sync.md

```diff
@@ -0,0 +1,35 @@
+# Pony Sync
+
+The Pony Sync is a weekly meeting where Pony contributors discuss topics related to Pony, including recently added or modified issues.
+
+## Running the Pony Sync
+
+The Pony Sync takes place as a [Zoom](https://zoom.us) call. In order to run the Pony Sync you must have Zoom installed, and be logged in as `ponylang.main@gmail.com`. A list of recently modified issues can be generated using the [pony-sync-helper]() application; this list should be pasted into the `sync` stream in Zulip, in a topic whose name is the date of the meeting in US date format (for example May 12, 2020 would be "5/12/2020").
+
+### Prerequisites
+
+In order to run the Pony Sync meeting you will need:
+
+1. A GitHub account
+1. A GitHub personal access token for running the `pony-sync-helper` program
+1. The [`pony-sync-helper`](https://github.com/aturley/pony-sync-helper) program
```

Comment:
Not directly related:

I think it makes sense to transition the sync helper to a ponylang org controlled project. Would you be open to that Andy?

Reply:
I agree. That would be fine. I'll do that and then update the links.

Reply:
Done.